server/api/locations/locations.api.js

```diff
@@ -15,54 +29,129 @@ exports.create = route(async (req, res) => {
   res.status(201).send(await serialize(req, location, policy));
 });
 
+/**
+ * Lists locations ordered by name.
+ *
+ * @function
+ */
+exports.list = route(async (req, res) => {
```

Comment:
I have doubts that this function will ever be used.

Reply:
We will use it once I've added the bbox filter to it. (And we'll probably add pagination in the future.)